workspaces/Makefile

```diff
@@ -0,0 +1,39 @@
+default:
+
+all: shared linux nitro
+
+shared:
+	$(MAKE) -C applications
+	$(MAKE) -C host
+
+linux: shared
+	$(MAKE) -C linux-host
+	$(MAKE) -C linux-runtime
+
+nitro: shared
+	$(MAKE) -C nitro-host
+	$(MAKE) -C nitro-runtime
+
+icecap: # shared
```

Comment:
If it's not needed, it should be removed, change the comment to "shared is not included because icecap uses its own environment to build applications and host"

Reply:
I have been discussing with @nspin, in this PR the applications and host use the `sdk-and-test-collateral` nix environment, but a later PR will change that.